packages/aws/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.42.1"
+  changes:
+    - description: Set event.type and event.action fields in vpcflow logs.
```

Since we need to change the version, let's also
```suggestion
    - description: Set `event.type` and `event.action` fields in vpcflow logs.
```